src/main/java/at/hannibal2/skyhanni/data/SackAPI.kt

```diff
@@ -328,6 +329,14 @@ object SackAPI {
         fetchSackItem(this).takeIf { it.statusIsCorrectOrAlright() }?.amount
 
     fun NEUInternalName.getAmountInSacks(): Int = getAmountInSacksOrNull() ?: 0
+
+    fun testSackAPI (args: Array<String>) {
+        if (args.size == 1 ) {
+            val amountInSacks = args[0].asInternalName().getAmountInSacksOrNull()
```

Comment:
Should check if it can go in sacks first before and if not send an error

Reply:
What do I use to do that?

Reply:
list in the repo

Reply:
SackAPi.sackListInternalNames.contains(args[0].asInternalname())